world/concert_with_human.sdf

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
```

Comment:
I cannot find the human in this file called `concert_with_human`, maybe I'm missing something.

Reply:
Line 12-15 are responsible to create the human